Integrations/KennaV2/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 ## [Unreleased]
+- Fixed an issue with search_vulnerability command while running on a list of id's"
+- Added the ability to run commands on a list of args of:
+    - ***id*** 
+    - ***status*** 
+    - ***top priorities*** 
```

Comment:
- Added the ability to run the commands ***kenna-***, ***kenna-***, ... with a list of the following arguments:
    - ***id*** 
    - ***status*** 
    - ***top priorities***